enterprise/server/cmd/ci_runner/main.go

```diff
@@ -860,6 +860,28 @@ func (ar *actionRunner) Run(ctx context.Context, ws *workspace) error {
 	// because that error is instead surfaced in the caller func when calling
 	// `buildEventPublisher.Wait()`
 
+	wfc := &bespb.WorkflowConfigured{
```

Comment:
nit: Could you add a comment that this has to be emitted first

Reply:
I have an in-progress PR that updates the BES server to not care about the order of these events, so will leave as-is